backends-common/redis/src/main/java/org/apache/james/backends/redis/RedisConfiguration.scala

```diff
@@ -104,11 +105,17 @@ case class StandaloneRedisConfiguration(redisURI: RedisURI, ioThreads: Option[In
 }
 
 object MasterReplicaRedisConfiguration {
-  def from(config: Configuration): MasterReplicaRedisConfiguration =
-    from(config.getStringArray("redisURL"),
+  def from(config: Configuration): MasterReplicaRedisConfiguration = {
+    val redisUris: Array[String] = config.getString(REDIS_URL_PROPERTY_NAME) match {
+      case rawValue if rawValue.startsWith("redis-sentinel:") => Array(config.getStringArray(REDIS_URL_PROPERTY_NAME).mkString(","))
```

Comment:
`config.getStringArray(REDIS_URL_PROPERTY_NAME).mkString(",")`

Why not simple config.getString(REDIS_URL_PROPERTY_NAME) ?

Reply:
I tried, but it didn't work

Reply:
ok